### PR TITLE
feat: add edge support 

### PR DESCRIPTION
**Pull request by kalifun**
Hi, I have finished testing `edge` in `win11`.
Edge Version：**109.0.1518.78**

I'll test it on `MacOs` tonight. If it doesn't work I will submit it again.

### CODE REVIEW

File: leetcode/client.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func NewClient(options ...ClientOption) Client {
 	httpClient := sling.New()
 	httpClient.Add(
 		"User-Agent",
-		"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.55 Safari/537.36",
+		"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.55 Safari/537.36 Edg/109.0.1518.78",
```
Comment (j178) [inline leetcode/client.go:98]: no need to change the UA I think ?

**Comment by j178** [comment]
Hi, thank you for opening this! Maybe you need to run go mod tidy to update the dependencies.

File: leetcode/client.go (hunk 1 above)
Comment (kalifun) [reply leetcode/client.go:98]: I added it to prevent anomalies.

**Comment by kalifun** [comment]
> Hi, thank you for opening this! Maybe you need to run go mod tidy to update the dependencies.

sorry, I fixed it right away

File: leetcode/client.go (hunk 1 above)
Comment (kalifun) [reply leetcode/client.go:98]: > no need to change the UA I think ?

I'll test this one later.

Comment (kalifun) [reply leetcode/client.go]: > no need to change the UA I think ?

yep,It's work. I have removed  `Edg/109.0.1518.78`

**Comment by j178** [comment]
thank you~
and there is one last thing, README_zh.md should be updated too. 😄

**Comment by kalifun** [comment]
> thank you~ and there is one last thing, README_zh.md should be updated too. 😄

😄Okay, sorry for dropping the ball.

**Pull request merged**